src/main/java/com/gtnewhorizons/angelica/hudcaching/HUDCaching.java

```diff
@@ -187,15 +190,15 @@ public static void fixGLStateBeforeRenderingCache() {
     }
 
     private static void resetFramebuffer(int width, int height) {
-        if (framebuffer.framebufferWidth != width || framebuffer.framebufferHeight != height) {
-        	framebuffer.createBindFramebuffer(width, height);
-            framebuffer.setFramebufferFilter(GL11.GL_NEAREST);
+        if (getFramebuffer().framebufferWidth != width || getFramebuffer().framebufferHeight != height) {
```

Comment:
nit: store the FB in a final local variable... it annoys me calling a getter every time.....